src/Containers/BlogPost.js

```diff
@@ -55,43 +59,15 @@ export default function BlogHome() {
         id
       }
     }
-  }
-  `
+  }`
+
   const [post, setPost] = useState([]);
-  const [addReaction, setAddReaction] = useState(false);
-  const [reactionCounter, setReactionCounter] = useState([]);
+  const [postNodeId, setPostNodeId] = useState('');
+  const [reactionPopup, setReactionPopup] = useState(false);
+  const [postReactions, setPostReactions] = useState([]);
   const { loading, error, data } = useQuery(GET_POSTS);
-
-  const getEmojiStringByName = useCallback((emojiName) => {
-    switch (emojiName) {
-      case "THUMBS_UP":
-        return "👍";
-
-      case "THUMBS_DOWN":
-        return "👎";
-
-      case "LAUGH":
-        return "😄";
-
-      case "HOORAY":
-        return "🎉";
-
-      case "CONFUSED":
-        return "😕";
-
-      case "HEART":
-        return "❤️";
-
-      case "ROCKET":
-        return "🚀";
-
-      case "EYES":
-        return "👀";
-
-      default:
-        return "";
-    }
-  }, []);
+  const reactionsContainer = useRef(null);
```

Comment:
Why useRef here?

Reply:
To force the reactions to refresh when adding/removing one

Reply:
OK, so we call API again to get the reactions or we update it manually

Reply:
Yes, I managed to use minimum API calls as possible in order to improve performances